kyuubi-server/src/main/scala/org/apache/kyuubi/server/trino/api/KyuubiTrinoOperationTranslator.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.server.trino.api
+
+import org.apache.hive.service.rpc.thrift.TProtocolVersion
+
+import org.apache.kyuubi.operation.OperationHandle
+import org.apache.kyuubi.service.BackendService
+import org.apache.kyuubi.session.KyuubiSessionManager
+import org.apache.kyuubi.sql.trino.TrinoGetSchemas
+
+class KyuubiTrinoOperationTranslator(backendService: BackendService) {
+  lazy val parser = backendService.sessionManager.asInstanceOf[KyuubiSessionManager].parser
```

Comment:
We aims to have another parser for transform operation with trino fe, in other words

1. reuse KyuubiSqlBaseLexer.g4 file.
2. new KyuubiTrinoFeBaseParser file.
3. make a KyuubiParserBase to abstract the parser

btw, in future, maybe need a new KyuubiMysqlFe parser to translate.

Reply:
it does not matter, we can have if necessary. For now, we can work with exists parser file first.